### PR TITLE
1677 download - renaming

### PR DESCRIPTION
**Pull request by emanuelcanha**
Rename package-manager to donwload-manager
Rename gmi.package to gmi.packages

### CODE REVIEW

File: src/demo-web-app/bundled-container/bundled/content/tools/picknmix-service/picknmix-service.js [hunk 1]
```diff
@@ -0,0 +1,396 @@
+var express = require('express')
```
Comment (ThomasCampbellBBC) [inline src/demo-web-app/bundled-container/bundled/content/tools/picknmix-service/picknmix-service.js]: Not sure why this is here. Suspect someone else already moved or deleted this file, and a merge has brought it back. Probably just needs deleting.

File: src/demo-web-app/bundled-container/bundled/content/tools/picknmix-service/responses/overrides.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[
```
Comment (ThomasCampbellBBC) [inline src/demo-web-app/bundled-container/bundled/content/tools/picknmix-service/responses/overrides.json]: Same as comment above

File: src/demo-web-app/bundled-container/bundled/content/tools/picknmix-service/picknmix-service.js (hunk 1 above)
Comment (emanuelcanha) [reply src/demo-web-app/bundled-container/bundled/content/tools/picknmix-service/picknmix-service.js]: Yes I had noticed it, was waiting for after the standup to delete it. It was one of my merges

**Comment by emanuelcanha** [comment]
Yeah I'm going to change the documentation now to:
- gmi.experiences
- gmi.packages

**Pull request merged**